examples/carla/manual_control.py

```diff
@@ -0,0 +1,595 @@
+#!/usr/bin/env python
+
+# Copyright (c) 2020 Computer Vision Center (CVC) at the Universitat Autonoma de
+# Barcelona (UAB).
+#
+# This work is licensed under the terms of the MIT license.
+# For a copy, see <https://opensource.org/licenses/MIT>.
+
+# Allows controlling a vehicle with a keyboard. For a simpler and more
+# documented example, please take a look at tutorial.py.
```

Comment:
Presumably this is referring to a file in CARLA? Would be good to add a full link so as not to confuse people.

Reply:
Removed